site/pages/react/login-methods/email-magic-link.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Email Magic Link Authentication
+description: How to implement Email Magic Link authentication in your React app
+---
+
+import ModalAuthExample from "../shared/modal-auth-example.mdx";
+import EmbeddedAuthExample from "../shared/embedded-auth-example.mdx";
+
+# Email Magic Link Authentication
+
+Email magic link authentication is a two-step process:
+
+1. The user enters their email address and requests a magic link
+2. The user clicks the link in their email, which redirects them back to your application to complete authentication
+
+:::tip
+Email OTP has been shown to have up to a 3x higher conversion rate and a 10-second faster flow compared to magic links. Consider using [Email OTP](/react/login-methods/email-otp) for better user experience.
+:::
+
+You can implement Email Magic Link authentication in two ways:
+
+- [Pre-built UI Components](#pre-built-ui-components) - Quick implementation with minimal code
+- [Custom UI](#custom-ui) - Complete control over the user experience
+
+## Pre-built UI Components
+
+Account Kit provides pre-built UI components that handle the entire Email Magic Link authentication flow with minimal code.
+
+### Step 1: Add Authentication Components to Your Page
+
+Before configuring your authentication, first add one of the pre-built components to your application:
+
+<ModalAuthExample />
+
+Or:
+
+<EmbeddedAuthExample />
+
+### Step 2: Configure Email Magic Link in UI Components
+
+After adding the components, configure the Email Magic Link authentication in your application config:
+
+```tsx
```

Comment:
for all of these codeblocks, can we add `twoslash` after the language? ie:
```suggestion
```tsx twoslash
```

this will ensure that any syntax bugs are caught now or in the future